backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -77,6 +77,11 @@ public UUID createShop(CreateShopRequest createShopRequest) {
 				.map(Map.Entry::getValue).collect(Collectors.toList());
 	}
 
+	public long getWaitingTime() {
+		long waitingTime = 2L;
+		return waitingTime;
+	}
```

Comment:
Think it is better to return waiting time wrt to the number of customers ahead in queue instead of returning a constant value for every customer.

Reply:
We will return a constant from backend to frontend for now. We'll multiply that with the required number of customers in the frontend and display the result there. (That was decided in the meeting with hosts, I had informed you about this in our call I guess.)